SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialReplicationGraph.cpp

```diff
@@ -17,7 +19,15 @@ void USpatialReplicationGraph::InitForNetDriver(UNetDriver* InNetDriver)
 	{
 		if (USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(InNetDriver))
 		{
-			LoadBalancingHandler = MakeUnique<FSpatialLoadBalancingHandler>(SpatialNetDriver);
+			bStrategyWorkerEnabled = USpatialStatics::IsStrategyWorkerEnabled();
+			if (bStrategyWorkerEnabled && SpatialNetDriver->LoadBalanceStrategy)
```

Comment:
Nit: Explicit check on LoadBalanceStrategy

Do we want to warn if bStrategyWorkerEnabled is enabled but LoadBalanceStrategy is null?

Reply:
+1

Reply:
Switched to an ensure

Reply:
Actually this isn't needed, so removed entirely.